pkg/static/login.css

```diff
@@ -86,6 +86,10 @@ button {
   opacity: .7;
 }
 
+.try-again {
```

Comment:
This can be done by giving the element `href="#"`. Or ideally using `<button>` and styling it as link.

Reply:
I opted for using `href="#"` styling as a button would introduce more custom css.